index.js

```diff
@@ -112,7 +112,8 @@ await axios
 
 const pollInterval = 3000; // in milliseconds
 
-let followWorkflow = getInput("Follow").toLowerCase() == "true";
+let followWorkflow =
```

Comment:
As a general practice, I would group input parsing at the top with the other uses of `getInput`.

You could then set `followWorkflow = false;` in the `catch` callback, similar to how you set it in the pollWorkflow's `catch` callback.